pyproject.toml

```diff
@@ -72,7 +72,7 @@ types-PyYAML = "^6.0.11"
 # production but only inside of the meadowrun container or the meadowrun AMI which we
 # fully control. We don't want to cause it to be downloaded when there's no need for it
 venv-pack = "^0.2.0"
-conda-pack = "^0.6.0"
+conda-pack = { git = "https://github.com/conda/conda-pack.git", tag="0.7.0"}
```

Comment:
Are we relying on conda-pack 0.7.0? If so we need to edit `docker_images/meadowrun/Dockerfile` (and theoretically build_image_shared.py as well, but we don't actually install conda-pack there because we don't use it)